data-sources/scanned-events-query-governance/config.yml

```diff
@@ -0,0 +1,9 @@
+id: scanned-events-query-governance
+displayName: Query Governance
+description: |
+  The dashboard provides real-time, interactive oversight of the volume of scanned events generated by queries from various sources. It consists of several key widgets that offer different perspectives on the query volume. The dashboard's dynamic and detailed nature facilitates a deep understanding of query volumes, empowering users to optimize queries and manage resources effectively.
+install:
+  primary:
+    link:
+      url: 
```

Comment:
Lets make sure we have a URL for this install definition. If it doesn't seem like there is a reasonable resource to point to here, lets omit the data source from the quickstart altogether.